UnitTests/UnitTests.HighPerformance.UWP/Package.appxmanifest

```diff
@@ -0,0 +1,46 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Package
+  xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
+  xmlns:mp="http://schemas.microsoft.com/appx/2014/phone/manifest"
+  xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
+  IgnorableNamespaces="uap mp">
+
+  <Identity Name="2db00f4f-0cf1-44bd-97ad-2bf93c49958d"
+            Publisher="CN=Sergio"
+            Version="1.0.0.0" />
+
+  <mp:PhoneIdentity PhoneProductId="2db00f4f-0cf1-44bd-97ad-2bf93c49958d" PhonePublisherId="00000000-0000-0000-0000-000000000000"/>
+
+  <Properties>
+    <DisplayName>UnitTests.HighPerformance.UWP</DisplayName>
+    <PublisherDisplayName>Sergio</PublisherDisplayName>
```

Comment:
```suggestion
    <PublisherDisplayName>Microsoft</PublisherDisplayName>
```

Reply:
Fixed in https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/3128/commits/760fcd2cdab1d8a0afe64056795d69b8324ca74a.